### PR TITLE
InsertUpdateDeleteHibertate

### PR DESCRIPTION
**Pull request by Yanetta**

### CODE REVIEW

**Review by Taniann — APPROVED** [review]
good

File: src/main/java/HibernateWork/HibOrderDaoImpl.java [hunk 1]
```diff
@@ -45,35 +70,110 @@ private static SessionFactory buildSessionFactory() {
...
     @Override
     public boolean updateOrder(Orders order) {
-        return false;
+        LOG.debug("update Orders instanse");
+        boolean isSuccessful = false;
+        try {
+            sessionObj = buildSessionFactory().openSession();
+            sessionObj.beginTransaction();
+            Orders orders = (Orders) sessionObj.load(Orders.class, order.getOrder_num());
+            orders.setQty(order.getQty());
```
Comment (SerhiiPasko) [inline src/main/java/HibernateWork/HibOrderDaoImpl.java]: And I don't see real operation on order to DB

File: src/test/java/hibernateTests/OrdersDaoImplTestHib.java [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+@FixMethodOrder(MethodSorters.NAME_ASCENDING)
+public class OrdersDaoImplTestHib {
+    private static final BigDecimal ALREADY_EXIST_ORDER = BigDecimal.valueOf(113042);
+
+    private HibOrderDaoImpl orderDa = new HibOrderDaoImpl();
+
+    private Orders order = new Orders(BigDecimal.valueOf(222223), null, null, null, "REI", null, BigDecimal.valueOf(500), BigDecimal.valueOf(600));
```
Comment (SerhiiPasko) [inline src/test/java/hibernateTests/OrdersDaoImplTestHib.java]: If you use same id - make sence to do it as constant

**Review by SerhiiPasko — CHANGES_REQUESTED** [review]
Read comments

**Review by SerhiiPasko — APPROVED** [review]
Good

**Pull request merged**